libs/data-transfer-objects/src/main/java/no/nav/testnav/libs/dto/pdlforvalter/v1/PersonDTO.java

```diff
@@ -251,4 +256,15 @@ public boolean isStatusIn(FolkeregisterPersonstatus... statuser) {
                         .build())
                 .getStatus());
     }
+
+    public Identtype getIdenttype() {
```

Comment:
Vil denne funke for test-norge eller nav-syntetiske identer?

Reply:
Nav syntetiske ja. 
testnorge-identer trolig ikke, men de er ikke i pdl-forvalter uansett. 😸

Reply:
Flott, ville bare sjekke 😄